pycode/memilio-epidata/memilio/epidata/cleanData.py

```diff
@@ -60,7 +68,7 @@ def clean_data(all_data, rki, john_hopkins, population, hdf5, out_path):
 
         # TODO: make general dictionary with all countries used
         directories = ['Germany/', 'Spain/', 'France/',
-                       'Italy/', 'US/', 'SouthKorea/', 'China/']
+                       'Italy/', 'US/', 'SouthKorea/', 'China/', 'Mobility/']
```

Comment:
The mobility file writes .txt files in data/mobility. Since the cleanData functionallity only cleanes .json and .h5 files, I deleted the 'mobility' folder. To discuss whether .txt files should be deleted in this functionallity or not. Maybe link this to #190?